java/src/jmri/jmrit/vsdecoder/VSDecoderManager.java

```diff
@@ -802,7 +802,7 @@ public void reporterPropertyChange(PropertyChangeEvent event) {
                             // look for additional geometric layout information
                             if (geofile_ok) {
                                 Reporter rp = (Reporter) event.getSource();
-                                int new_rp = Integer.parseInt(rp.getSystemName().substring(2));
+                                int new_rp = Integer.parseInt(Manager.getSystemSuffix(rp.getSystemName()));
```

Comment:
This will also throw an exception if system suffix is not an integer. But that would have happen before this change too.

Reply:
Done. Thanks.